frontend/src/components/common/OtherCommunitie.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Row, Card, CardImg, CardBody, CardTitle, CardText, CardLink, Button } from "reactstrap";
+
+type OtherCommunitieProps = {
```

Comment:
```suggestion
type OtherCommunitiesProps = {
```

Reply:
OtherCommunityProps instead

And on line 15, we'll have
`const OtherCommunity = ({ description, github_account, image, name, others_link, telegram, twitter, website }: OtherCommunityProps) => {`